codeship/setup.sh

```diff
@@ -3,5 +3,5 @@
 # Exit script with error if any step fails.
 set -e
 
-npm install -g serverless
+npm install -g serverless@3.7
```

Comment:
Is it advantageous to lock it to a specific minor version like that, rather than just to the major version?

Reply:
Honestly, I can't remember why I chose to do that.